### ISSUE TITLE
Accessibility-> Missing Skip to main content link when using the keyboard.

### ISSUE THREAD

**Issue by manuji** [bcgov/nr-rec-resources#591]
**Describe the bug**
For users relying on keyboard interface, there should be means of skipping repeating content. so that users can move around pages efficiently. This issue fails the [2.4.1 Bypass Blocks (Level A)](https://www.w3.org/WAI/WCAG22/Understanding/bypass-blocks) criterion.

**To Reproduce**
1. Go to 'https://d1vrz2gt4nmfe5.cloudfront.net/'
2. Using the Keyboard only try navigating through the Beta site.
3. Check if you can skip the header section and skip to the main content of the page easily. 

**Expected behavior**
There should be an option to skip the repetitive content on a page and navigate to the main content of the page. 

**Screenshots**

https://github.com/user-attachments/assets/c33c2ec3-8d40-4289-bc57-a193ec8952f9

**Comment by jimmypalelil** [comment]
@airliah @harryaolson @jaimiebutton 

is this about adding a new link somewhere in the page for easier navigation? do we need mockups for this one?

**Comment by manuji** [comment]
Hi @jimmypalelil , usually the "Skip to main" content has to be places at the beginning of the tabbing order of a page. Here is an example from the bcparks.ca. Thanks

<img width="483" alt="Image" src="https://github.com/user-attachments/assets/4bcd0840-828c-47d0-b67c-8f975184b8e2" />

**Comment by airliah** [comment]
@harryaolson, @manuji, @jaimiebutton, @marcellmueller , @jimmypalelil , @fbarreta and @BAAlexK can you review this and determine the expected behavior? Currently what I'm seeing as the fix is that is automatically skips over the header content but then that doesn't give users the choice? do we need to have actually UI here that 'skips'? would love to know what is recommended here and confirm if our fix is good or we need/want to add anything? thanks!

**Comment by airliah** [comment]
this is partially fixed and will be launched tomorrow, however there are still some issues that are written up in a new ticket: https://github.com/bcgov/nr-rec-resources/issues/785

**Comment by manuji** [comment]
Verified that a link to skip the header has been added, but when you are using the keyboard only, the "Skip to main content" link does not show focus style indications. 

<img width="702" alt="Image" src="https://github.com/user-attachments/assets/afd0628a-bd28-4805-9d57-482e26d2b4da" />

FYI @airliah
